README.md

```diff
@@ -41,59 +43,57 @@ Rocky is an framework for building powerful and scalable APIs for your imp-power
-- *timeout* &mdash; Modifies how long Rocky will hold onto a request before automatically executing the *onTimeout* handler
+If your code doesn’t alter Rocky’s default behavior, you still need to call *init()*.
 
-These are the default settings:
+All of Rocky’s methods are accessible as before, and return the same values. *init()* returns a reference to the Rocky singleton. There is no longer a distinction between class and instance methods: all of Rocky’s methods can be called on Rocky itself, or an alias variables, as these reference the same table:
 
 ```squirrel
-defaults <- {
-    accessControl = true,
-    allowUnsecure = false,
-    strictRouting = false,
-    timeout = 10
-}
+// These calls are equivalent
+app.get("/users/([^/]*)", function(context) {
+    local username = context.matches[1];
+});
+
+Rocky.get("/users/([^/]*)", function(context) {
+    local username = context.matches[1];
+});
 ```
 
-<div id="rocky_verb"><h3>VERB(<i>signature, callback[, timeout]</i>)</h3></div>
-The *VERB()* methods allow you to assign routes based on the specified verb and signature. The following *VERB*s are allowed:
+**Note** [Rocky.Context](#context) and [Rocky.Route](#route) continue to be implemented as classes, but remember that you will not be creating instances of these classes yourself &mdash; new instances will be made available to you as needed, by Rocky.
 
-- app.get(*signature, callback[, timeout]*)
-- app.put(*signature, callback[, timeout]*)
-- app.post(*signature, callback[, timeout]*)
+#### Initialization Options ####
 
-When a match is found on the verb (as specified by the method) and the *signature*, the callback function will be executed. The callback takes a [Rocky.Context](#context) object as a parameter. An optional route-level timeout can be passed in. If no timeout is passed in, the timeout set in the constructor will be used.
+A table containing any of the following keys may be passed into *init()* to modify the library’s default behavior:
 
-```squirrel
-// Responds with '200, { "message": "hello world" }'
-// when the user makes a GET request to the agent URL:
-app.get("/", function(context) {
-    context.send({ "message": "hello world" })
-})
-```
+| Key | Description |
+| --- | --- |
+| *accessControl* | Modifies whether Rocky will automatically add `Access-Control` headers to the response object. Default: `true` |
+| *allowUnsecure* | Modifies whether Rocky will accept HTTP requests. Default: `false` (ie. HTTPS only) |
+| *strictRouting* | Enables or disables strict routing. Default: `false` (ie. Rocky will consider `/foo` and `/foo/` to be identical) |
+| *sigCaseSensitive* | Enforce [signature](#signatures) case sensitivity. Default: `false` (ie. Rocky will consider `/FOO` and `/foo` to be identical) |
+| *timeout* | Modifies how long Rocky will hold onto a request before automatically executing the *onTimeout* handler. Default: 10s |
+
+<div id="signatures"><h4>Signatures</h4></div>
```

Comment:
signatures are not introduces yet and are not used during initialization. should this be moved to after verb and on methods?

Reply:
For info that doesn't relate to a specific method, style is to include the info in the usage section. IMHO, the sig section gets a bit lost down in among the methods, and it's relevant here because we've introduced *sigCaseSenstive* in the initialisation options. it should have a different headline weight (3 not 4), so I will change that.